Gemfile

```diff
@@ -22,6 +22,10 @@ gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
+# Database handler
+gem 'pg', '~> 0.18.4'
+# Devise for user and authentication handling
+gem 'devise'
```

Comment:
Add gem version. This is because if you install the project in the future, if a major change happens in the gem, you are not affected and everything keeps working in the same way.